### PR TITLE
copy-edit-skill-success-criteria-for-loops

### LINKED ISSUE #459: https://github.com/Umuzi-org/ACN-syllabus/tree/develop/content/specific-skill-success-criteria/for-loops/_index.md

### PR DESCRIPTION
**Pull request by asradebe**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: content/specific-skill-success-criteria/for-loops/_index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ for x in range(len(arr))
     ...
 ```
 
-- They should be able to move forwards or backwards.
+- They should be able to move forward or backwards.
```
Comment (JacobUmuzi) [inline content/specific-skill-success-criteria/for-loops/_index.md:38]:
```suggestion
- They should be able to move forward or backward.
```

File: content/specific-skill-success-criteria/for-loops/_index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Many people do pattern matching instead of understanding. Here are some common t
 
 ## Note about submission format
 
-On Tilde you'll notice that this card is asking for a link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
+On Tilde you'll notice that this card is asking for link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
```
Comment (JacobUmuzi) [inline content/specific-skill-success-criteria/for-loops/_index.md]:
```suggestion
On Tilde you'll notice that this card is asking for a link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
```

**Review by JacobUmuzi — CHANGES_REQUESTED** [review]

File: content/specific-skill-success-criteria/for-loops/_index.md (hunk 1 above)
Comment (asradebe) [reply content/specific-skill-success-criteria/for-loops/_index.md:38]: lol, the sentence doesn't sound right

File: content/specific-skill-success-criteria/for-loops/_index.md (hunk 2 above)
Comment (asradebe) [reply content/specific-skill-success-criteria/for-loops/_index.md]: I removed it cause Grammarly throws an error(indefinite article may be redundant when used with the uncountable noun submission)

**Review by JacobUmuzi — APPROVED** [review]

**Pull request merged**